Description: Fix get_release_by_discid
 The includes allowed for /ws/2/discid are the same as for /ws/2/release.
Origin: upstream, https://github.com/alastair/python-musicbrainz-ngs/commit/4260ed
Author: Wieland Hoffmann <themineo@gmail.com>
Forwarded: yes
Last-Update: 2013-01-15

--- a/musicbrainzngs/musicbrainz.py
+++ b/musicbrainzngs/musicbrainz.py
@@ -64,8 +64,11 @@
 		"release-group-rels", "url-rels", "work-rels"
 	],
 	'discid': [
-		"artists", "labels", "recordings", "release-groups", "puids",
-		"echoprints", "isrcs"
+		"artists", "labels", "recordings", "release-groups", "media",
+		"artist-credits", "discids", "puids", "echoprints", "isrcs",
+		"artist-rels", "label-rels", "recording-rels", "release-rels",
+		"release-group-rels", "url-rels", "work-rels", "recording-level-rels",
+		"work-level-rels"
 	],
 	'echoprint': ["artists", "releases"],
 	'puid': ["artists", "releases", "puids", "echoprints", "isrcs"],
@@ -631,8 +634,8 @@
 
 
 # Lists of entities
-def get_releases_by_discid(id, includes=[], release_type=[]):
-	params = _check_filter_and_make_params(includes, release_type=release_type)
+def get_releases_by_discid(id, includes=[], release_status=[], release_type=[]):
+	params = _check_filter_and_make_params(includes, release_status, release_type=release_type)
 	return _do_mb_query("discid", id, includes, params)
 
 def get_recordings_by_echoprint(echoprint, includes=[], release_status=[], release_type=[]):
